tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -107,10 +107,9 @@ spec:
               # Calculate the source container image based on the provided container image
               sourceContainer="${source_repo}:${source_reference_tag}"
 
-              sourceContainerDigest=$(skopeo inspect \
-                --no-tags \
-                --format '{{.Digest}}' \
-                "docker://${sourceContainer}" 2>/dev/null)
+              AUTH_FILE=$(mktemp)
+              select-oci-auth "${sourceContainer}" > "$AUTH_FILE"
+              origin_digest=$(oras resolve --registry-config "$AUTH_FILE" "${sourceContainer}" 2>/dev/null)
```

Comment:
is this supposed to set `sourceContainerDigest` instead of `origin_digest`? The change removes `sourceContainerDigest` and I don't see `origin_digest` used elsewhere (but I do see `sourceContainerDigest` used)

Reply:
nice catch. leftover from a copy/paste from push_snapshit it seems. fixed